community-code/binance-trading-neptune-master/notebooks/experiment.ipynb

```diff
@@ -0,0 +1,1414 @@
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32m[I 2022-06-20 17:27:22,235]\u001b[0m Trial 16 finished with value: 3.783288091847585 and parameters: {'n_estimators': 351, 'max_depth': 6, 'learning_rate': 0.03465490975710614, 'subsample': 0.7551852317711564, 'colsample_bytree': 0.6023129836271461, 'gamma': 13, 'look_back': 97}. Best is trial 13 with value: 5.210315443125857.\u001b[0m\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "8f3ec982f9d74002bcf3d7bd368775b9",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "  0%|          | 0/35064 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32m[I 2022-06-20 17:28:30,557]\u001b[0m Trial 17 finished with value: 1.2864036246032193 and parameters: {'n_estimators': 465, 'max_depth': 4, 'learning_rate': 0.09979112701397053, 'subsample': 0.823579011259816, 'colsample_bytree': 0.733666759764732, 'gamma': 17, 'look_back': 161}. Best is trial 13 with value: 5.210315443125857.\u001b[0m\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "9c4f28dd2b8442a68ab1ecf3ece61327",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "  0%|          | 0/35064 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32m[I 2022-06-20 17:29:27,501]\u001b[0m Trial 18 finished with value: 5.029711072029103 and parameters: {'n_estimators': 444, 'max_depth': 6, 'learning_rate': 0.05395073296801107, 'subsample': 0.8754804974592673, 'colsample_bytree': 0.8524895396223497, 'gamma': 17, 'look_back': 66}. Best is trial 13 with value: 5.210315443125857.\u001b[0m\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "eee5993f539240a2a6ff33ffe30d1aa5",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "  0%|          | 0/35064 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[32m[I 2022-06-20 17:31:10,770]\u001b[0m Trial 19 finished with value: 4.373678616655161 and parameters: {'n_estimators': 832, 'max_depth': 6, 'learning_rate': 0.05645751761800897, 'subsample': 0.8949849334127633, 'colsample_bytree': 0.8418850562727114, 'gamma': 17, 'look_back': 70}. Best is trial 13 with value: 5.210315443125857.\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "import optuna\n",
+    "import warnings\n",
+    "import neptune.new as neptune\n",
```

Comment:
@BexTuychiev / @domikoz - this is the legacy `neptune-client` API: https://docs.neptune.ai/setup/upgrading/

Are we still using this in blogs?

Reply:
@BexTuychiev please make sure everything is up-to-date 🙏

Reply:
I made another commit changing all instances of outdated code. Also, updated the dependencies